configurations/dockerFileTemplate.go

```diff
@@ -8,32 +8,32 @@ import (
 )
 
 type dockerfileTemplate struct {
-	Data serverConfig
+	Data config
 }
 
 // Gives a new instance of dockerfile template.
-func newDockerfileTemplate(data serverConfig) *dockerfileTemplate {
+func newDockerfileTemplate(data config) *dockerfileTemplate {
 	return &dockerfileTemplate{Data: data}
 }
 
-// Unmarshal yaml file and gives an instance of serverConfig.
-func newDockerFileDataFromYamlFile(filename string) (serverConfig, error) {
+// Unmarshal yaml file and gives an instance of config.
+func newDockerFileDataFromYamlFile(filename string) (config, error) {
 	node := yaml.Node{}
 
 	err := unmarshalYamlFile(filename, &node)
 	if err != nil {
 		return nil, fmt.Errorf("unmarshal: %v", err)
 	}
 
-	serverConfigData, err := getConfigsDataFromNode(node.Content[0])
+	configData, err := getConfigsDataFromNode(node.Content[0])
 	if err != nil {
-		return nil, fmt.Errorf("can't extract server config from node: %v", err)
+		return nil, fmt.Errorf("can't extract config from node: %v", err)
 	}
-	return serverConfigData, nil
+	return configData, nil
 }
 
-// Returns the serverConfig instructions provided in yaml.
-func getConfigsDataFromNode(node *yaml.Node) (serverConfig, error) {
+// Returns the config instructions provided in yaml.
+func getConfigsDataFromNode(node *yaml.Node) (config, error) {
 	var data dockerfileDataYaml
 
 	err := verifyConfigYamlNode(node)
```

Comment:
hard to tell what this function does. What is node?

Reply:
Node is the yaml node being read

Reply:
Yeah. I'm not entirely for parsing yaml files this way. I have raised an issue for this.

Reply:
Yes. Will fix it in next PR